src/core/net/nd6.hpp

```diff
@@ -505,6 +506,119 @@ class RaFlagsExtOption : public Option, private Clearable<RaFlagsExtOption>
 
 static_assert(sizeof(RaFlagsExtOption) == 8, "invalid RaFlagsExtOption structure");
 
+/**
+ * Represents the NAT64 Prefix Information Option.
+ *
+ * See section 4 of RFC 8781 for definition of this option [https://tools.ietf.org/html/rfc8781#section-4]
+ */
+OT_TOOL_PACKED_BEGIN
+class Nat64PrefixInfoOption : public Option, private Clearable<Nat64PrefixInfoOption>
```

Comment:
Although combining the general idea of a **Prefix Information Option** with the specific context of **NAT64** leads to the correct concept, but I'm wondering if **NAT64 Prefix Option** or **PREF64 Option** is more commonly used and adopted. 

Note that there is no **NAT64 Prefix Information Option** presented in RFC 8781.